features/comment.feature

```diff
@@ -152,8 +154,9 @@ Feature: Manage WordPress comments through the REST API
     When I try `wp rest comment delete 1`
     Then STDERR should contain:
       """
-      Error: Sorry, you can not delete this comment
+      Error: Sorry, you are not allowed to delete this comment. HTTP code: 401
       """
+    Then the return code should be 145
```

Comment:
I'm still not sure how I feel about using these elevated return codes (WP-CLI still uses `0` vs `1`). It doesn't need to block the PR though; we can discuss separately #96